README.md

```diff
@@ -11,7 +19,38 @@ import numpy as np
 x_sample = np.random.normal(size = (500, 10))
 y_sample = np.random.normal(size = (400, 10))
 
+# To get pvalues from PQMass
 pvalues = pqm_pvalue(x_sample, y_sample, num_refs = 100, bootstrap = 50)
-
 print(np.mean(pvalues), np.std(pvalues))
+
+# To get chi2 from PQMass
+chi2_stat = pqm_chi2(x_sample, y_sample, num_refs = 100, bootstrap = 50)
+print(np.mean(chi2_stat), np.std(chi2_stat))
+```
+
+If your two samples are drawn from the same distribution then the pvalue should
+be drawn from the random uniform(0,1) distribution. This means that if you get a
+very small value (i.e. 1e-6) then you have failed the null hypothesis test and
+the two samples are not drawn from the same distribution.
+
+For the chi2 metric, given your two sets of samples, if they come from the same
+distribution, the histogram of your chi² values should follow the chi² distribution. 
```

Comment:
Could you just use `chi^2` instead of the superscript? It's hard to read and may not display correctly on all devices (its even smaller on my phone)